docs/_docs/integrations/badges.md

```diff
@@ -8,28 +8,76 @@ order: 10
 Dependency-Track supports badges in Scalable Vector Graphics (SVG) format. Support for badges is a globally configurable
 option and is disabled by default.
 
+> Enabling badge support will provide vulnerability and policy violation metric information to unauthenticated users.
+> Any anonymous user with network access to Dependency-Track and knowledge of a projects information will be able
+> to view the SVG badge.
+
+In all following examples, replace `{name}`, `{version}`, and `{uuid}` with their respective values.
+
+### Vulnerable components
+Create a badge for vulnerable components of the project. It either shows...
+
+* ... the severity of the vulnerabilities.
+* ... "no vurns" if there are no vulnerabilities.
+* ... "no metrics" if metrics for the project aren't collect yet.
```

Comment:
```suggestion
Create a badge for vulnerable components of the project. It either shows:

* the severity of the vulnerabilities.
* "no vulns" if there are no vulnerabilities.
* "no metrics" if metrics for the project aren't collect yet.
```

Reply:
Corrected. You have right my ... would have been a break in style. And i better learn to spell vulnerabilities correct ;)